includes/onnxruntime/onnxruntime_cxx_inline.h

```diff
@@ -7,6 +7,8 @@
 // These are the inline implementations of the C++ header APIs. They're in this separate file as to not clutter
 // the main C++ file with implementation details.
 
+#pragma once
```

Comment:
not needed.

Reply:
done

Reply:
I prefer to keep it that way. When python crashes, there is no way to tell which test failed unless we keep that verbosity.